contents/blog/best-open-source-ab-testing-tools.md

```diff
@@ -0,0 +1,165 @@
+---
+date: 2022-05-24
+title: The 5 best free and open-source A/B testing tools
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/experiments.jpeg
+featuredImageType: full
+categories: ["Open source", "Guides"]
+author: ["hanna-crombie"]
+---
+
+Developers and marketers are customer experts, so it’s tempting to assume we can instinctively make the best decisions for user engagement. This is a trap.
+ 
+Leaders don’t want to make changes without evidence and however much we think we know about our users, making changes on theory alone is fraught with danger. That’s why you need an A/B testing tool.
+
+Also known as split testing or multivariate testing, A/B testing is the implementation of experiments which split your audience to test variations of a product design, new feature, call to action, landing page – anything you can imagine, really.
```

Comment:
I would personally cut the first two paras and just open with the third one ('Also known as split testing....'). The first two paras don't add any value, and anyone searching for this already will be convinced enough that they need an A/B testing tool.

Reply:
Fair. This is already considerably cut down from the original verision, but I'm always in favour of jumping right in.